planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -46,6 +45,7 @@
     public static final String REGION = "region";
     public static final String HARDWARE_ID = "hardwareId";
     public static final String TYPE = "type";
+    public static final String ENDPOINT = "endpoint";
     public static final String CLOUD_FOUNDRY = "CloudFoundry";
     public static final String POLICIES = "policies";
     public static final String GROUPS = "groups";
```

Comment:
can you please add a javadoc to the generateDAM method?
this will help particulalry to know what is inside the String returned, thanks